vignettes/beers-vignette.Rmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "beers"
+author: "Wes Hinsley"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The Beers interpolation is used particularly in demographic interpolation, where
+frequently it is necessary to interpolate 5-yearly data, down to single year points.
+Standard UNWPP population data for instance, is 5-yearly in its native representation,
+but UNWPP use the Beers ordinary method (among other transformations) to produce
+the interpolated data.
+
+## Installation
```

Comment:
Don't include installation instructions in the vignette - by the time they're reading that they have already installed the package.  It's fine in the README though.  But the installation instructions here are not great because they won't work until the package is on CRAN

Reply:
Progress so far... 
1a. R6 removed and turned into closures!
1b. Closures removed and turned into functions!
2. Tests added.
3. I'm using travis - I think. Got more reading to do on that.
4. (not done yet...)